dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/metadata/objectbundle/DefaultObjectBundleService.java

```diff
@@ -78,7 +79,7 @@ public class DefaultObjectBundleService implements ObjectBundleService {
   private final CurrentUserService currentUserService;
   private final PreheatService preheatService;
   private final SchemaService schemaService;
-  private final SessionFactory sessionFactory;
+  @PersistenceContext private EntityManager entityManager;
```

Comment:
does this get instantiated correctly given that it's not marked as `final`? Which is what the `RequiredArgsConstructor` uses for the class constructor.

Reply:
nice catch. This was when i haven't found away to inject using constructor. Fixed now.